tests/multiparser_js_markdown/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -1,5 +1,36 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`0-indent.js 1`] = `
+md\`
+This line shouldn't be indented at all in the resulting output.
+\`
+
+if (true) {
+  md\`
+text1
+
+text2
+
+text3
+\`;
+}
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+md\`
+This line shouldn't be indented at all in the resulting output.
+\`;
+
+if (true) {
+  md\`
+text1
+
+  text2
+
+  text3
+  \`;
```

Comment:
I was just gonna comment this too :)

I think we need to split and join with `literalline`s, like in https://github.com/prettier/prettier/blob/dc4e44968f430e54eaa72fb55752dc78fd8df755/src/language-js/printer-estree.js#L1917-L1918

Reply:
Do we have `literal-line` or `literal-softline`? The linebreaks in markdown is not static. 🤔

Reply:
I've made a `zeroIndent` primitive to print zero-indented docs. Do you want me to open a separate PR or push here?

Reply:
I'll open a PR refactoring the template literal printing